integration/docker/entrypoint.sh

```diff
@@ -67,18 +71,22 @@ done
 
 case ${service,,} in
   master)
-    bin/alluxio format
+    if [[ ${options} != ${NO_FORMAT} ]]; then
```

Comment:
(nit) It would be nice to print some error message if options does not match any valid value

Reply:
Thanks